### PR TITLE
Adding a sample for fluent validation

### PR DESCRIPTION
**Pull request by indualagarsamy**
Sample for https://docs.particular.net/nservicebus/messaging/validation-fluentvalidation

### CODE REVIEW

File: samples/fluent-validation/FluentValidation_1/Endpoint/Endpoint.Core.csproj [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <OutputType>Exe</OutputType>
+    <LangVersion>7.1</LangVersion>
+    <ApplicationIcon />
+    <StartupObject />
```
Comment (SimonCropp) [inline samples/fluent-validation/FluentValidation_1/Endpoint/Endpoint.Core.csproj]: `<ApplicationIcon />    <StartupObject />` seem redundant?

File: samples/fluent-validation/FluentValidation_1/Endpoint/Endpoint.Core.csproj [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="NServiceBus" Version="7.*" />
+    <PackageReference Include="NServiceBus.FluentValidation" Version="1.1.1" />
```
Comment (SimonCropp) [inline samples/fluent-validation/FluentValidation_1/Endpoint/Endpoint.Core.csproj]: should be a `*` dependency?

File: samples/fluent-validation/FluentValidation_1/Endpoint/MyHandler.cs [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+using System.Threading.Tasks;
+using NServiceBus;
+
+class MyHandler :
+    IHandleMessages<MyMessage>
+{
+    public Task Handle(MyMessage message, IMessageHandlerContext context)
+    {
+        Console.WriteLine($"Hello from MyHandler. MyMessage content is: {message.Content}");
+        return Task.FromResult(0);
```
Comment (SimonCropp) [inline samples/fluent-validation/FluentValidation_1/Endpoint/MyHandler.cs]: `Task.completedtask`

File: samples/fluent-validation/FluentValidation_1/Endpoint/Program.cs [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        configuration.UseTransport<LearningTransport>();
+
+        #region Enable
+        var validation = configuration.UseFluentValidation(outgoing: false);
+        validation.AddValidatorsFromAssemblyContaining<MyMessage>();
+        #endregion
+
+        var endpoint = await Endpoint.Start(configuration);
+
+        await endpoint.SendLocal(new MyMessage {Content = "sd"});
```
Comment (SimonCropp) [inline samples/fluent-validation/FluentValidation_1/Endpoint/Program.cs]: isnt config await the standard for docs?

**Comment by indualagarsamy** [comment]
Decided to not add an outgoing validation to this sample, for the sake of simplicity at this time. We are already doing local sends to show the validation during the message handling process.

**Pull request merged**